CHANGELOG-1.0.md

```diff
@@ -0,0 +1,138 @@
+# Release notes for 
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v0.5.0
+
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Do not call controller-expand if volume is in-use and can not be expanded while in-use. New RBAC rules for listing pods are required. ([#86](https://github.com/kubernetes-csi/external-resizer/pull/86), [@gnufied](https://github.com/gnufied))
+- Adds a new flag `handle-volume-inuse-error` which can be set to `false` if a CSI driver does not want in-use check to be performed before issuing CSI expansion RPC calls([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc))
```

Comment:
For drivers that do support online resizing, should they set it to false?

Reply:
I have reworded this to:

>This behaviour is enabled by default but can be disabled by setting `handle-volume-inuse-error` to `false`, which would allow external-resizer to perform volume expansion without checking if volume being expanded is in-use or not([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc)).

I would leave this to k8s admin to decide how they want to configure it, may be?

Reply:
sorry, I ended up adding a note about online resizing.